src/signals/incident/components/form/DescriptionInfo/DescriptionInfo.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
+import { makeSelectSubCategories } from 'models/categories/selectors';
+import { breakpoint } from '@datapunt/asc-ui';
+import { fetchCategories } from 'models/categories/actions';
+import { makeSelectIncidentContainer } from 'signals/incident/containers/IncidentContainer/selectors';
+import { isAuthenticated } from 'shared/services/auth/auth';
+
+const DescriptionInfoStyle = styled.div`
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+
+  @media screen and ${breakpoint('max-width', 'laptopM')} {
+    flex-direction: column;
+  }
+
+  & > :first-child {
+    flex-grow: 1;
+    white-space: nowrap;
+  }
+`;
+
+const DescriptionInfo = ({ info }) => {
+  const dispatch = useDispatch();
+  const subcategories = useSelector(makeSelectSubCategories);
+  const { subcategoryPrediction } = useSelector(makeSelectIncidentContainer);
+  const [suggestion, setSuggestion] = useState();
+
+  useEffect(() => {
+    if (!subcategories && isAuthenticated()) {
+      dispatch(fetchCategories());
+    }
+  }, [subcategories, dispatch]);
```

Comment:
You should not have to request subcategories in a component deep down in the application. I would move this call to the `App` container component and remove the call from the settings module, since the categories list is retrieved there as well.

Reply:
Is done in another PR.